### PR TITLE
(BLOG) Status Update 2021-04-23

### PR DESCRIPTION
**Pull request by da-ar**

### CODE REVIEW

File: _posts/2021-04-23-status-update.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+  - status
+  - puppet-camp
+  - litmus
+  - bolt
+  - facter
+  - cloud-ci
+  - rsapi
+  - devx
+  - dsc
+tags:
```
Comment (sanfrancrisko) [inline _posts/2021-04-23-status-update.md:16]: Could do with filling out the tags - I'd nearly say what's in categories should be in `tags` and keep `categories`: `status`, `team`

File: _posts/2021-04-23-status-update.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  [ruby-pwsh-pr-118]: https://github.com/puppetlabs/ruby-pwsh/pull/118
+  [Hvid]: https://github.com/Hvid
+  [rspec-puppet-pr-823]: https://github.com/rodjek/rspec-puppet/pull/823
+  [tiandrey]: https://github.com/tiandrey
+  [rspec-puppet-pr-819]: https://github.com/rodjek/rspec-puppet/pull/819
+  [ekohl]: https://github.com/ekohl
+
+## puppet_litmus is now compatible with Bolt 3.x
+
+A massive shout out to https://github.com/beechtom and the Bolt team for working on testing puppet_litmus for Bolt 3.x support. It has now been released and is compatible with Bolt 3.x
```
Comment (sanfrancrisko) [inline _posts/2021-04-23-status-update.md:139]:
```suggestion
A massive shout out to [Tom Beech](https://github.com/beechtom) and the Bolt team for working on testing puppet_litmus for Bolt 3.x support. It has now been released and is compatible with Bolt 3.x
```

File: _posts/2021-04-23-status-update.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+We have also started the investigation for the porting of [puppetlabs-sqlserver][puppetlabs-sqlserver] module.
+We have updated all our multi node modules to use the new default litmus inventory file location, wrt the latest changes in the litmus [GH-380](https://github.com/puppetlabs/puppet_litmus/issues/380).
+From this week we started to work on preparing training and educational materials for cloud CI.
+
+## Resource API: Custom Insync
+
+We've been doing some work over the last couple of weeks to make it possible to [add custom insync handling to Resource API resources][rsapi-custom-insync-issue].
+In short, this is to help with cases where the default property comparison logic in Puppet doesn't work for a use case - when you want to compare case insensitive strings or only ensure that _some_ members of a group exist (without having to specify _all_ members of that group) for example.
+We're nearing completion of that work, wich you can see in this [draft PR][rsapi-custom-insync-draft-pr].
```
Comment (sanfrancrisko) [inline _posts/2021-04-23-status-update.md:157]:
```suggestion
We're nearing completion of that work, which you can see in this [draft PR][rsapi-custom-insync-draft-pr].
```

**Review by sanfrancrisko — COMMENTED** [review]
Just minor comments, but otherwise LGTM 👍 

Good job on the 1st blog post!

**Review by DavidS — COMMENTED** [review]
chunky one :+1:

**Review by sanfrancrisko — APPROVED** [review]

**Pull request merged**